source/client-side-encryption/etc/test-templates/aggregate.yml.template

```diff
@@ -27,14 +27,12 @@ tests:
       - command_started_event:
           command:
             listCollections: 1
-            cursor: {}
```

Comment:
Note, C driver does not send a cursor document for listCollections if no batch size is specified. I did not see in the [enumerate collections spec](https://github.com/mongodb/specifications/blob/master/source/enumerate-collections.rst) any requirement to do so. So I removed this from expectations.